### ISSUE TITLE
Option to strip "window." when unnecessary

### ISSUE THREAD

**Issue by TrevorBurnham** [terser/terser#561]
**Feature request**

It'd be nice if Terser were smart enough to recognize when the global namespace object is being used unnecessarily (given some combination of options that makes this case unambiguous). Example:

**`terser` input**

window.addEventListener('resize', () => { console.log('resize'); });

**Desired `terser` output**

```js
addEventListener("resize",()=>{console.log("resize")});
```

**Comment by fabiosantoscode** [comment]
This is a pretty good suggestion. A lot of lints force people to use window.location and window.navigator.

It's not a hard thing to implement. It does need an extra option though but that should be OK.

Thanks for the feature request!

**Comment by jridgewell** [comment]
I think this is actually an unsafe, unless we're very conservative on values that are allowed. Eg, `addEventListener` would throw if we run in node.

**Comment by TrevorBurnham** [comment]
True, it would change behavior. But if we’re talking about direct calls to window.addEventListener, then it’d just be changing one type of error to another.
